### ISSUE TITLE
clip edit in GL viewer is still broken

### ISSUE THREAD

**Issue by gaponenko** [root-project/root#13418]
### Check duplicate issues.

- [ ] Checked for duplicates

### Description

Hello,

It looks like  #13081 is closed, but the problem is still there.   I compiled root from today's head of the master branch, c15efc8fb313595a018245f112adf3f30c014776.      The only difference from my earlier report is that one needs to use the --web=off switch to the command line to make the image appear in the GL viewer window (where the box clip is available) instead of a browser window (which does not even have the box clip).     Once the "edit in viewer" option is checked, the clip box can be moved with the mouse, but still only along one axis, not along the other two.

This issue may be specific to the newest version of Alma Linux.

Andrei

### Reproducer

To reproduce: run

$ root --web=off $ROOTSYS/tutorials/geom/building.C

- in the GL window, click on the "Clipping" tab
- click "Box"
- check "Edit in Viewer"
- rotate to make the three clip box axes visible
- try to move the clip box along the axes. It moves along one axis, but not along the other two.

### ROOT version

```
------------------------------------------------------------------
| Welcome to ROOT 6.29/01                        https://root.cern |
| (c) 1995-2022, The ROOT Team; conception: R. Brun, F. Rademakers |
| Built for linuxx8664gcc on Aug 08 2023, 14:22:00                 |
| From heads/master@v6-29-01-2303-gc15efc8fb3                      |
| With c++ (GCC) 11.3.1 20221121 (Red Hat 11.3.1-4)                |
| Try '.help'/'.?', '.demo', '.license', '.credits', '.quit'/'.q'  |
------------------------------------------------------------------
```
### Installation method

build from source

### Operating system

AlmaLinux 9.2

**Comment by couet** [comment]
I tried again on my Mac with master and I can move it along the 3 axes. I do not have an AlmaLinux machine to test.

**Comment by gaponenko** [comment]
Hi Olivier,

Does CERN have any AlmaLinux servers that you can login to?  I see for example nightly builds for EL9, /cvmfs/sft.cern.ch/lcg/views/dev3/latest/*el9*
It looks to me like the problem goes with the machine where one runs ROOT, not with the machine which is used for display, so you may be able to ssh from your laptop to a server to reproduce the problem.

Thank you for your reply,
Andrei

________________________________________

**Comment by couet** [comment]
Yes, i am sure the problems come from the machine on which ROOT runs. I asked @bellenot who told me lxplus9 might run this OS. I ll try

**Comment by couet** [comment]
Gl does not seem properly installed there: 
```
$ glxgears
libGL error: No matching fbConfigs or visuals found
libGL error: failed to load driver: swrast
Error: glXCreateContext failed
```

**Comment by gaponenko** [comment]
Hi Olivier,

I had a similar situation with a Fermilab server and requested computing support to install missing libraries.  I do not have a CERN account so can't open a ticket for lxplus9, but it makes sense to install the libraries there to make the machine usable with ROOT.

Andrei

________________________________________

**Comment by couet** [comment]
Yes indeed. I opened a ticket for that.

**Comment by couet** [comment]
As you might see the the ticket mentioned above I have no clue how to display a GL application coming from lxplus. So I am stuck and cannot test your problem. sorry for that.

**Comment by ferdymercury** [comment]
I tried with Linux over SSH -X, and after installing libftgl-dev, everything worked fine.

@gaponenko could you crosscheck in recent versions of ROOT and AlmaLinux? With:

root --web=off $ROOTSYS/tutorials/visualisation/geom/building.C

**Comment by ferdymercury** [comment]
Considering that it is not a ROOT related issue and that there is https://cern.service-now.com/service-portal?id=ticket&table=incident&n=INC3561368, I would propose closing this. Please reopen Andrei if you think it's needed. Thanks.

**Comment by gaponenko** [comment]
I confirm this functionality works on AlmaLinux now.

Thank you,
Andrei

________________________________________
